core/provider.proto

```diff
@@ -37,7 +37,7 @@ message RegisterArgs {
 }
 
 message RegisterRet {
-  ErrorNumber errno = 1;
```

Comment:
??? give me a reason.

Reply:
`errno` will be generated literally by `protoc`, which is a macro in cpp however